src/edit_python_pe/markdown_builder.py

```
@@ -0,0 +1,283 @@
+    path_md = os.path.join(
+        member_app.repo_path,
+        BLOG_DIR,
+        MEMBERS_DIR,
+        filename,
+    )
+
+    if not os.path.exists(path_md):
+        return
+    try:
+        content = _read_file(path_md)
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)